src/integration/endpoints.ts

```diff
@@ -4,6 +4,7 @@ export enum ServiceNames {
   'ros-backend' = 'ros-backend',
   'chrome-service' = 'chrome-service',
   'advisor-backend' = 'advisor',
+  'vulnerability-engine' = 'vulnerability',
```

Comment:
Can you change it to vulnerability engine here?

```suggestion
  'vulnerability-engine' = 'vulnerability-engine',
```